### ISSUE TITLE
Illumination Mode: sanitize arguments

### ISSUE THREAD

**Issue by demisjohn** [demisjohn/ASML_JobCreator#44]
User tried to change Illumination mode (to the default of `conventional`), but conversion produce the following errors:

```
Reticle Data - Layer ID (NMESA) Image ID (NMESA5V1)
The 'lithography process ' (Conventional) is invalid.
```

for every Layer.

He may have had to guess at the text to input.  Maybe as simple as needing ALL-CAPS in the exported TXT file.  Check allowed values.  Ultimate method is to BIN-->TXT convert jobs that are set to various Illum. modes, and see what the ASML outputs.

**Comment by fyodr** [comment]
Layer._parse_IllumMode should be sanitizing already...? It forces "Default" or "Conventional" or "Annular" in output. Apparently only "Default" is allowed by the conversion program? Have other users tried to control the illumination mode here?

This issue may require some debugging at the tool.

**Comment by demisjohn** [comment]
That’s right - I just guessed in the code, but it turns out we need to do the following:
- Generate a binary job at the tool, that uses Annular illum (and maybe another for “conventional”).
- Convert the binary job to text.
- inspect the resulting text file to see what the string is supposed to be, or if any additional fields showed up. 
- modify export code accordingly. 

I apologize I should have commented that code to state that it was untested.
